pkg/dao/repository_config_test.go

```diff
@@ -1016,7 +1016,7 @@ func (suite *RepositoryConfigSuite) TestValidateParameters() {
 	assert.False(t, response.URL.Skipped)
 	assert.Contains(t, response.URL.Error, "blank")
 
-	mockExtRDao.Mock.On("ValidRepoMD", "http://foobar.com").Return(200, nil)
+	mockExtRDao.Mock.On("ValidRepoMD", "http://foobar.com/").Return(200, nil)
```

Comment:
http://foobar.com/ returns 403 not 200
Try example.com if you want 200 responce code.

Reply:
I can change this, but the URL doesn't need to return anything here. The 200 response code is being mocked. In other words, this line means "return 200 for this URL".